src/mono/System.Private.CoreLib/src/System/TypeSpec.cs

```diff
@@ -326,7 +326,7 @@ private static void SkipSpace(string name, ref int pos)
         private static void BoundCheck(int idx, string s)
         {
             if (idx >= s.Length)
-                throw new ArgumentException("Invalid generic arguments spec", "typeName");
+                throw new ArgumentException(SR.Argument_InvalidGenericArg);
```

Comment:
```suggestion
                throw new ArgumentException(SR.Argument_InvalidGenericArg, "typeName");
```

For the `ArgumentException` constructors, the second value is the name of the argument the exception is for. This should be retained.

Reply:
Ok ill retain the 2nd parameter for the argumentExceptions in the next commit

Reply:
Does the new commit fix this problem?

Reply:
Yes. Thank you.